networking/rocket.capnp

```diff
@@ -0,0 +1,26 @@
+interface Rocket {
```

Comment:
What is this file for?

Reply:
it's irrelevant, we used it for demonstration purposes for the Network module